src/components v2/Atoms/Forms/CheckboxGroup/CheckboxGroup.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <P2Checkbox
+      v-for="(checkbox, index) in items"
+      :key="checkbox.label"
+      class="checkbox-group-item"
+      :size="size"
+      :disabled="disabled"
+      :name="`${name}-${checkbox.label}`"
+      :value="checkbox.value"
+      :label="checkbox.label"
+      @input="$emit('input', { index, value: $event })"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import P2Checkbox from '../Checkbox';
+
+export default Vue.extend({
+  name: 'CheckboxGroup',
+
+  components: {
+    P2Checkbox,
+  },
+
+  props: {
+    items: {
+      type: Array,
+      default: (): [] => [],
```

Comment:
Is this meant to be this `() => []` ? That `:` colon mean to be there?

Reply:
Good point. I think this was legacy and I just copied it. I never really stood still by the weirdness of the structure.

Doesn't help that github copilot also plonks it in :D